packages/flutter_tools/templates/plugin/lib/projectName_platform_interface.dart.tmpl

```diff
@@ -0,0 +1,27 @@
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import '{{projectName}}_method_channel.dart';
```

Comment:
Nit: Add a blank line above this one to separate package imports from local imports.